modules/compute/gke-node-pool/variables.tf

```diff
@@ -255,6 +255,12 @@ variable "timeout_update" {
   default     = null
 }
 
+variable "user_workload_path" {
+  description = "The path to the user workload, this should point to the kubernetes job manifest that user want to have the TCPX sidecar injected."
```

Comment:
nit: Should we point customers to the external guide (or our code) which can give them an example ?

Reply:
https://cloud.google.com/kubernetes-engine/docs/how-to/gpu-bandwidth-gpudirect-tcpx#add-gpudirect-manifests

Reply:
updated